go/enclave/rollupchain/rollup_chain.go

```diff
@@ -840,6 +894,22 @@ func (rc *RollupChain) produceRollup(b *types.Block, bs *obscurocore.HeadsAfterL
 	// Postprocessing - withdrawals
 	txReceiptsMap := toReceiptMap(txReceipts)
 	r.Header.Withdrawals = rc.bridge.RollupPostProcessingWithdrawals(r, newRollupState, txReceiptsMap)
+	crossChainMessages, err := rc.crossChainProcessors.LocalManager.ExtractLocalMessages(txReceipts)
+	if err != nil {
+		rc.logger.Crit("[CrossChain] Extracting messages L2->L1 failed", err)
```

Comment:
general comment. 
The convention with this logging mchanism is that you can go to log.go and add a new Cmp.
Let's say `CrossChainCmp="CrossChain`, 
and then replace the [CrossChain] with:
`rc.logger.Crit("Extracting messages L2->L1 failed",log.ErrKey, err, log.CmpKey, log.CrossChainCmp)`

Btw, you're missing the `log.ErrKey`